cmd/helm/search.go

```diff
@@ -58,6 +61,10 @@ func searchChartRefsForPattern(search string, chartRefs map[string]*repo.ChartRe
 
 func searchCacheForPattern(dir string, search string) ([]string, error) {
 	fileList := []string{}
+	if _, err := os.Stat(dir); err != nil {
+		return fileList, errors.New("No cache exists. Did you run 'helm init'?")
```

Comment:
I think that we'd have to run the entire `helm init` process, which doesn't seem like a good idea to do here. Thoughts?